README.md

```diff
@@ -2,6 +2,8 @@
 
 [![Build Status](https://travis-ci.com/stripe/stripe-cli.svg?token=eQWDVpt1sJR63TxbC1KA&branch=master)](https://travis-ci.com/stripe/stripe-cli)
 
+_🏗 The Stripe CLI is currently in beta! We're working on more features to make the experience great. If you have any [feedback](https://docs.google.com/forms/d/e/1FAIpQLScA6MwRBmfyWSBRzxUPd_P2oy9uL6_bFLjOUsnRRYiwAUDCXw/viewform), find [issues](https://github.com/stripe/stripe-cli/issues), or would like to be involved in more active testing, please let us know!_
```

Comment:
Maybe use our redirect link instead? Just in case we need to change the redirect at some point.

```suggestion
_🏗 The Stripe CLI is currently in beta! We're working on more features to make the experience great. If you have any [feedback](https://stri.pe/cli-feedback), find [issues](https://github.com/stripe/stripe-cli/issues), or would like to be involved in more active testing, please let us know!_
```

Reply:
Good call, updating!